src/systems/ships/ship_movement.rs

```diff
@@ -16,12 +15,8 @@ pub fn ai_movement(
 ) {
     characters
         .par_iter_mut()
-        .for_each(|(mut transform, mut ai_ship)| {
-            let ship_speed = ai_ship.current_velocity * time.delta_seconds();
-
-            ai_ship.current_velocity = (ai_ship.current_velocity + ai_ship.acceleration)
```

Comment:
Took away the initial AI acceleration because it is mostly redundant. May add some docking AI or something in the future that requires slowing down or something like that.